tests/model_serving/conftest.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional, Tuple
```

Comment:
You should also add fixtures for the buckets in this conftest

Reply:
code moved to tests.conftest